src/test/scala/temple/utils/StringUtilsTest.scala

```diff
@@ -24,4 +24,34 @@ class StringUtilsTest extends FlatSpec with Matchers {
     indent("abcd\nefg", 1) shouldEqual " abcd\n efg"
     indent("abcd\nefg\n", 1) shouldEqual " abcd\n efg\n "
   }
+
+  behavior of "snakeCase"
```

Comment:
What should a table CustomerID be rendered as? Can that be enforced with tests?

Reply:
There was a test where this was checked implicitly, but I've added a more explicit one to make that clear:

https://github.com/TempleEight/temple/pull/83/files#diff-5a2f0a5251b6239394f7b57213f42947R39